sailfishusermanager.cpp

```diff
@@ -287,6 +288,18 @@ uint SailfishUserManager::addUser(const QString &name)
     return uid;
 }
 
+int SailfishUserManager::removeUserFiles(const char *user)
+{
+     struct passwd *pwd = getpwnam(user);
+     if (pwd) {
+         QDir dir(USER_ENVIRONMENT_DIR.arg(pwd->pw_uid));
+         if (dir.removeRecursively())
+             return 0;
+         qCWarning(lcSUM) << "Removing user environment directory failed";
+     }
+     return 1;
```

Comment:
Logic wise this would look like a "bool function()", but as it is actually exit code -> using EXIT_SUCCESS / EXIT_FAILURE would be cleared than literal 0 / 1.